examples/web-wagmi/src/authentication/LoginSpecificProfile.tsx

```diff
@@ -18,95 +10,31 @@ function UnauthenticatedContent() {
     <>
       <h2>Login to a Specific Profile</h2>
 
-      <p>Handle</p>
-
-      <input
-        placeholder="@me"
-        onChange={(e) => {
-          setHandle(e.target.value);
-        }}
-        value={handle}
-        style={{ display: 'block' }}
-      />
+      <label>
+        Full handle (without `@` suffix)
+        <input
+          placeholder="lensprotocol"
+          onChange={(e) => {
+            setHandle(e.target.value);
+          }}
+          value={handle}
+          style={{ display: 'block' }}
+        />
+      </label>
 
       <LoginButton handle={handle} />
     </>
   );
 }
 
-function AuthenticatedContent({
-  wallet,
-  profile,
-}: {
-  wallet: WalletData;
-  profile: ProfileOwnedByMe;
-}) {
-  const { data: profiles } = useProfilesOwnedBy({ address: wallet.address });
-  const [handle, setHandle] = useState<string>();
-
-  const { execute: login } = useWalletLogin();
-  const { execute: logout } = useWalletLogout();
-
-  const { isConnected } = useAccount();
-  const { disconnectAsync } = useDisconnect();
-
-  const { connectAsync } = useConnect({
-    connector: new InjectedConnector(),
-  });
-
-  const onLoginClick = async () => {
-    await logout();
-    if (isConnected) {
-      await disconnectAsync();
-    }
-
-    const { connector } = await connectAsync();
-
-    if (connector instanceof InjectedConnector) {
-      const walletClient = await connector.getWalletClient();
-      await login({
-        address: walletClient.account.address,
-        handle: handle,
-      });
-    }
-  };
-
-  return (
-    <>
-      <h2>Login to a Specific Profile</h2>
-
-      <p>
-        You are already logged into a profile; Pick a different one below to log into a different
-        one after logging out.
-      </p>
-
-      <select
-        onChange={(e) => {
-          setHandle(e.target.value);
-        }}
-        value={handle || profile.handle}
-      >
-        {profiles?.map((p) => {
-          return (
-            <option key={p.handle} value={p.handle}>
-              {p.handle}
-            </option>
-          );
-        })}
-      </select>
-
-      <button onClick={onLoginClick}>
-        <strong>Log in</strong>
-      </button>
-    </>
-  );
-}
-
```

Comment:
I simplified this example... when already logged-in, logging-in with another handle on the same wallet is not a use case. One should use `useActiveProfileSwitch` for that purpose.